cmd/kubeconform/main.go

```diff
@@ -46,6 +49,26 @@ func processResults(cancel context.CancelFunc, o output.Output, validationResult
 	return result
 }
 
+func parseStdin(stdin *os.File) *strings.Reader {
+	var content string
+
+	// Read all stdin lines.
+	scanner := bufio.NewScanner(stdin)
+	for scanner.Scan() {
+		content += fmt.Sprintln(scanner.Text())
+	}
+
+	// Check if the stdin input is ResourceList.
+	rl, err := fn.ParseResourceList([]byte(content))
+	// If it's ResourceList, use the items as the new content.
+	if err == nil {
+		content = rl.Items.String()
```

Comment:
That's one part I'm worried about... there are some ridiculous optimisations to ensure we read stdin as a stream, and from previous tickets it looks like some people do indeed pass very large amounts of data via stdin... if we were to do that, this function would have to operate on a stream without buffering the whole thing